content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,140 @@
+
+**tagging** - [Tags][12] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][13] to manage and analyze them.
+
+## Setting up an integration
+
+The Datadog Agent package includes most integrations, so you don't need to download each individually.
+
+### API and Application keys
+
+In order to download the the Datadog Agent package, you need an [API and Application key][14]. You can manage your accounts API and Application keys in the [API Settings page][15] of the UI.
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][16] for specific instructions on how to connect. For most other integrations, start by installing the [Datadog Agent][17]. Most integrations are supported on our containerized agents: [Docker][18], and [Kubernetes][19]. After you've downloaded the Agent, go to the [Integrations page][16] section to find specific configuration instructions for individual integrations.
+
+### Configuration
+
+Configurations are more specifc to [individual integrations][16]. Configure most integrations by editing the `conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory. Most integration have a sample `conf.yaml` that lists all available configuration options.
+
+For example, this is a sample `conf.yaml` for the `http_check` integration:
+
+```
+init_config:
+
+instances:
+  - name: Example website
+    url: https://example.com/
+    # disable_ssl_validation: false      # default is true, so set false to check SSL validation
+    # ca_certs: /path/to/ca/file         # e.g. /etc/ssl/certs/ca-certificates.crt
+    # check_certificate_expiration: true # default is true
+    # days_warning: 28                   # default 14
+    # days_critical: 14                  # default 7
+    # timeout: 3                         # in seconds. Default is 10.
+  - name: Example website (staging)
+    url: http://staging.example.com/
+```
+
+### Validation
+
+To validate your configuration, [run the Agent's `status` subcommand][20], and look for new configuration under the Checks section.
+
+### Tagging
+
+Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. You can assign tags in configuration files, in environment variables, in the UI, or the API, in DogStatsD, and new integrations automatically inherited existing tags. For more information about tagging, see [Getting started with tags][12].
+
+## Installing multiple integrations
+
```

Comment:
This part should go in the integrations configuration for Agent based integration.